### ISSUE TITLE
Custom filter params

### ISSUE THREAD

**Issue by duongtdvn** [Zizzamia/ng-tasty#158]
Hi guys,

Is there any way to use the custom params for filter. For example, my backend allow params like this:
<pre>
http://domain.com/shop/products?search=name:tasty t-shirt;
</pre>
or
<pre>
http://domain.com/shop/products?search=tasty t-shirt&fields=name;
</pre>
those are same as the current ng-tasty filter:
<pre>
http://domain.com/shop/products?name=tasty t-shirt;
</pre>

Thank you!

**Comment by Zizzamia** [comment]
Why you have space on the url?

**Comment by duongtdvn** [comment]
Yep, it's name of the product I want to search and well, it will be encoded by the browser, some thing like:
<pre>
http://www.domain.com/shop/products?search=tasty%2520t-shirt&fields=name%3b
</pre>

I just want to make it more readable xD.
